ompi/errhandler/errhandler_predefined.c

```diff
@@ -148,7 +148,7 @@ void ompi_mpi_errors_return_win_handler(struct ompi_win_t **win,
 
 static void out(char *str, char *arg)
 {
-    if (ompi_mpi_initialized && !ompi_mpi_finalized) {
+    if (ompi_rte_initialized && !ompi_mpi_finalized) {
```

Comment:
@rhc54 Will `ompi_rte_initialized` automatically be false once we get far enough in MPI_FINALIZE?  I.e., do we still need to check `!ompi_mpi_finalized`?